Aries/Aries/Admin/Credentials/AdminCredentials.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  AdminCredentials.swift
+//  Aries
+//
+//  Created by David Clawson on 12/7/21.
+//
+
+import Foundation
+
+
+public class AdminCredentials {
+    
+    private var messageSender: MessageSender
+    private var adminConnection: ConnectionRecord
+    
+    public init(messageSender: MessageSender, adminConnection: ConnectionRecord){
+        self.messageSender = messageSender
+        self.adminConnection = adminConnection
+    }
+    
+    /**
+     Sets the instance's admin connection.
+      - Parameters:
+        - adminConenction: URL for the invitation.
+     - Returns:
+        - void
+     */
+    private func setAdminConnection(adminConenction: ConnectionRecord){
+        self.adminConnection = adminConenction
+    }
+    
+    /**
+     Sends a ReceiveInvitation message to the admin connection to start a connection process.
+     - Parameters:
+       - None.
+    - Returns:
+       - GetCredentialsListMessage.
+     */
```

Comment:
```suggestion
    /**
     Sends a ReceiveInvitation message to the admin connection to start a connection process.
    - Returns:
       - GetCredentialsListMessage.
     */
```
I don't think we should have the Parameters category if there are no parameters.

Reply:
Yeah, I missed that. Cool!